src/assets/styles/reset.scss

```diff
@@ -0,0 +1,78 @@
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default padding */
+ul[class],
+ol[class] {
+  padding: 0;
+}
+
+/* Remove default margin */
```

Comment:
it is better to write *{
margin: 0;
padding: 0;
} and connect this to index.js or inside index.html
It will provide less style code

Reply:
I think it should be a specific selector list. These styles don't need to be for all elements.

Reply:
Please avoid using * in your styles